src/viam/sdk/config/resource.cpp

```diff
@@ -110,7 +110,7 @@ ResourceConfig ResourceConfig::from_proto(viam::app::v1::ComponentConfig proto_c
     }
 
     if (proto_cfg.has_frame()) {
-        LinkConfig lc = LinkConfig::from_proto(proto_cfg.frame());
+        resource.frame_ = LinkConfig::from_proto(proto_cfg.frame());
```

Comment:
Is this diff tested?

Reply:
Good question! I don't think this is actually tested anywhere at the moment. I'll make a ticket for adding those tests.